matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/user/accountdata/UpdateBreadcrumbsTask.kt

```diff
@@ -41,6 +42,8 @@ internal class DefaultUpdateBreadcrumbsTask @Inject constructor(
 ) : UpdateBreadcrumbsTask {
 
     override suspend fun execute(params: UpdateBreadcrumbsTask.Params) {
+        // Do not add local rooms to the recent rooms list as they should not be known by the server
+        if (RoomLocalEcho.isLocalEchoId(params.newTopRoomId)) return
```

Comment:
Good point, but maybe this could be filtered before, at https://github.com/vector-im/element-android/blob/c73c36eb26149390167d23944b9fa3264cbc10f4/matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/DefaultRoomService.kt#L175, WDYT?

Reply:
Done https://github.com/vector-im/element-android/pull/7154/commits/648498e2de4e8af16b8a576c576c3f6737771812
I was hesitating between both places, I choose the task to be sure it would never be sent to the server but the task is only called by the service until now